GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/GVFSUpgradeReminderTests.cs

```diff
@@ -82,6 +89,35 @@ public void UpgradeTimerScheduledOnServiceStart()
             timerScheduled.ShouldBeTrue();
         }
 
+        private void ReadNugetConfig(out string feedUrl, out string feedName)
+        {
+            ProcessResult result = ProcessHelper.Run("gvfs", "config upgrade.feedurl");

```

Comment:
Rather than adding the calls to `gvfs config` in this test fixture, could you add helper method(s) to [GVFSProcess.cs](https://github.com/Microsoft/VFSForGit/blob/master/GVFS/GVFS.FunctionalTests/Tools/GVFSProcess.cs)?

That's where we've been adding the code for calling the other VFS4G verbs during the functional tests.